ReactNativeClient/lib/components/shared/reduxSharedMiddleware.js

```diff
@@ -28,6 +28,21 @@ const reduxSharedMiddleware = async function(store, next, action) {
 		refreshTags = true;
 	}
 
+	if (action.type === 'NOTE_SELECT' ||
+		action.type === 'NOTE_SELECT_TOGGLE') {
+		let noteTags = [];
+		// We don't need to show tags unless only one note is selected.
+		if (newState.selectedNoteIds && newState.selectedNoteIds.length === 1) {
```

Comment:
@laurent22 - I've made this fairly restrictive since we only display note contents when a single tag is selected.